### PR TITLE
Add trivial scripted tests

### PR DESCRIPTION
**Pull request by TonioGela**
I added some trivial scripted tests to the template and the CI for that.
We might need more parts of sbt-typelevel to add the mergify configuration before running scala-steward against this repo

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,10 +1,10 @@
-// This build is for this Giter8 template.
-// To test the template run `g8` or `g8Test` from the sbt session.
-// See https://www.foundweekends.org/giter8/testing.html#Using+the+Giter8Plugin for more details.
 lazy val root = (project in file("."))
-  .enablePlugins(ScriptedPlugin)
```
Comment (TonioGela) [inline build.sbt:5]: https://github.com/foundweekends/giter8/issues/860#issuecomment-1918127995

File: src/main/g8/$script_name;format="Camel"$.scala [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//> using toolkit typelevel:default
+//> using toolkit typelevel::default
```
Comment (zetashift) [inline src/main/g8/$script_name;format="Camel"$.scala:1]: Ah nice catch!

**Review by zetashift — APPROVED** [review]

**Comment by zetashift** [comment]
Whoops totally glossed over for this:
>We might need more parts of sbt-typelevel to add the mergify configuration before running scala-steward against this repo

I would like to have scala-steward support here, but entirely unsure where to start. I never worked with setting it up.

**Comment by TonioGela** [comment]
> I would like to have scala-steward support here, but entirely unsure where to start. I never worked with setting it up.

We need Arman for this, as he has all the needed permissions. It should be as simple as installing the TL Scala steward Github application on this repo too.

To be helpful we probably need to switch to explicit dependency versions, though.

**Pull request merged**